src/main/kotlin/app/configuration/JobConfiguration.kt

```diff
@@ -24,7 +25,7 @@ class JobConfiguration {
 
     @Bean
     fun importUserJob(step: Step) =
-        jobBuilderFactory.get("snapshotSenderJob")
+        jobBuilderFactory.get("ucHistoricDataImporterJob")
```

Comment:
Why is the Encryption processing being removed from everywhere? Is it being added back elsewhere later?

Reply:
The encrypted stream will go through a series of transformations by sequence of item processors (decryption processor , decompression processors and produce a input stream which will then be materialised  by the message creation processor to save messages to base